products/sql/terraform.yaml

```diff
@@ -39,6 +39,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       charset: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  SourceRepresentationInstance: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_source_representation_instance_basic"
+        primary_resource_id: "instance"
+        vars:
+          name: "my-instance"
+    properties:
+      onPremisesConfiguration: !ruby/object:Overrides::Terraform::PropertyOverride
+        flatten_object: true
+      onPremisesConfiguration.host: !ruby/object:Overrides::Terraform::PropertyOverride
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validateIpAddress'
+      onPremisesConfiguration.port: !ruby/object:Overrides::Terraform::PropertyOverride
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validation.IntBetween(0, 65535)'
+      region: !ruby/object:Overrides::Terraform::PropertyOverride
+        required: false
+        default_from_api: true
+        description: |
+          The Region in which the created address should reside.
```

Comment:
nit - maybe created address for the instance? just to make it slightly less confusing/so it looks less like a typo

Reply:
oh no definitely a typo, all fixed now